src/notebooklm/cli/helpers.py

```
@@ -76,6 +76,51 @@ def run_async(coro):
+    """Delete all sources with error status from a notebook.
+
+    Returns the number of deleted sources.
+    """
+    try:
+        sources = await client.sources.list(notebook_id)
+        error_ids = [s.get("id") for s in sources if s.get("status") == "error" and s.get("id")]
+        for source_id in error_ids:
+            try:
+                await client.sources.delete(notebook_id, source_id)
+            except Exception:
+                pass
```

Ruff (0.15.7)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)